style.css

```diff
@@ -1,18 +1,652 @@
-	font-family: "Montserrat-Medium";
+font-family: "Montserrat-Medium";
 }
 .parameters__small{
-	font-family: "Montserrat-Regular";
+font-family: "Montserrat-Regular";
+}
+
+
+
+
+
+
+
+
+
+body{
+max-width: 1440px;
+border: 1px solid orange;
+margin: 0;
+display: flex;
+flex-direction: column;
+}
+
+@media (max-width:1440px){
+	body{
+	max-width: 1400px;
+	}
+	.header{
+		max-width: 1400px;
+	}
+}
+
+@media (max-width:992px){
+	body{
+	max-width: 768px;
+	}
+	.header{
+		max-width: 768px;
+	}
+}
+
+@media (max-width:767px){
+	body{
+	max-width: 375px;
+	}
+	.header{
+		max-width: 375px;
+	}
+}
+
+.header{
+border: 1px solid orange;
+display: flex;
+max-width: 1440px;
+height: 51px;
+padding-top: 21px;
+justify-content: center;
+align-content: center;
+align-items: center;
+justify-content: space-between;
+}
+
+.header__title{
+	border: 1px solid orange;
+	padding-left: 41px;
+	margin-right: 250px;
+}
+
+.menu{
+border: 1px solid blue;
+font-size: 1vw;
+display: flex;
+justify-content: center;
+align-content: center;
+/*width: max-content;*/
+}
+
+.menu__store{
+	display: flex;
+	justify-content: center;
+	align-content: center;
+	bottom: 20px;
+	border: 1px solid green;
+	margin-left: 160px;
+}
+
+.menu__link{
+	border: 1px solid red;
+	min-width: 65px;
+display: flex;
+align-items: center;
+align-content: center;
+font-weight: 600;
+font-size: 15px;
+line-height: 18px;
+text-decoration: none;
+cursor: pointer;
+color: black;
+transition: color 0.3s ease-in;
+margin-right: 27px;
+}
+
+.menu__link_red{
+	border-bottom: 2px solid red;
+	display: flex;
+	align-items: center;
+	align-content: center;
+	font-weight: 600;
+	font-size: 15px;
+	cursor: pointer;
+	color: #E51937;
+	margin-right: 63px;
+	padding-bottom: 6px;
+	padding-top: 6px;
+}
+
+.menu__link_roadster{
+	margin-left: 20px;
+}
+
+.menu__link_energy{
+	margin-left: 15px;
+}
+
+.menu__link:hover{
+color: #E51937;
+}
+
+.header__burger{
+border: 2px solid green;
+display: flex;
+align-items: center;
+justify-content: flex-start;
+bottom: 60px;
+width: 24px;
+height: 20px;
+cursor: pointer;
+margin-right: 60px;
+}
+
+.header__burger span{
+width: 24px;
+height: 4px;
+background-color: #E51937;
+}
+
+.header__burger span:before{
+	position: absolute;
+	content: " ";
+	width: 24px;
+	height: 4px;
+	background-color: #E51937;
+	transform: translateY(-9px);
+}
+
+.header__burger span:after{
+	position: absolute;
+	content: " ";
+	width: 24px;
+	height: 4px;
+	background-color: #E51937;
+	transform: translateY(9px);
+}
+
+
+section{
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+}
+
+.start{
+display: flex;
+align-items: center;
+}
+
+.start__title{
+border: 1px solid green;
+display: flex;
+width: 370px;
+height: 90px;
+margin: 55px 0 0 0;
+font-style: normal;
+font-weight: 600;
+font-size: 90px;
+}
+
+.start__text{
+border: 1px solid green;
+margin-top: 10px;
+font-style: normal;
+font-weight: 500;
+font-size: 15px;
+line-height: 18px;
+position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)